src/snap/plugin.cc

```diff
@@ -52,7 +53,46 @@ Plugin::Meta::Meta(Type type, std::string name, int version) :
                      concurrency_count(5),
                      exclusive(false),
                      cache_ttl(std::chrono::milliseconds(500)),
-                     strategy(Strategy::LRU) {}
+                     strategy(Strategy::LRU),
+                     tls_enabled(false),
+                     tls_certificate_key_path(""),
+                     tls_certificate_crt_path(""),
+                     tls_certificate_authority_path(""),
+                     tls_cipher_suites("") { 
+                       const char* tls_env = getenv("TLS_ENABLED");
+                       if (tls_env != nullptr && strcmp(tls_env, "false") != 0) {
+                         std::stringstream message_template{};
+                         message_template << "Encryption enabled but ";
+
+                         const char* taoCS = getenv("TAOCS");
+                         const char* taoCC = getenv("TAOCC");
+                         const char* taoCK = getenv("TAOCK");
+                         const char* taoCA = getenv("TAOCA");
+
```

Comment:
Awesome - I figured these changes would need to be made after that merge! Thanks for the headsup.